model-optimizer/extensions/front/tf/softplus_ext.py

```diff
@@ -0,0 +1,26 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.front.extractor import FrontExtractorOp
+
+
+class SoftPlusExtractor(FrontExtractorOp):
+    op = 'Softplus'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        return cls.enabled
```

Comment:
Implement new Op Softplus and implement exctractors for Softplus for operations from other frameworks (ONNX and MxNet).